tests/commands/run_test.py

```diff
@@ -974,6 +974,12 @@ def test_pass_filenames(
 
 
 def test_fail_fast(cap_out, store, repo_with_failing_hook):
+    ret, printed = _do_run(
+        cap_out, store, repo_with_failing_hook, run_opts(fail_fast=True),
+    )
+    # it should have only run one hook
+    assert printed.count(b'Failing hook') == 1
```

Comment:
- this should be a separate test
- this test doesn't fail on master (it's 1 failing hook independent of fail_fast)

Reply:
Amended.